src/pages/games/secret-hitler/Form/index.tsx

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import { Box, Button, Typography } from "@material-ui/core";
+import { Form } from "react-final-form";
+import { FieldArray } from "react-final-form-arrays";
+import arrayMutators from "final-form-arrays";
+import { Radios } from "mui-rff";
+
+import PlayerRow from "./PlayerRow";
+import validate from "./validate";
+import { SHFormProps, FormData } from "./types";
+
+const initialValues = { players: [null, null, null, null, null] };
+
+const FormComponent = () => <SHForm initialValues={initialValues} />;
+
+const SHForm: React.FC<SHFormProps> = (props) => {
+  const { initialValues } = props;
```

Comment:
Instead of using the wrapper component, use the default values instead: https://github.com/reactjs/rfcs/blob/createlement-rfc/text/0000-create-element-changes.md#deprecate-defaultprops-on-function-components.

```suggestion
const FormComponent: React.FC<SHFormProps> = ({ initialValues = initialValues }) => {
```

Reply:
Oh my, it broke things spectacularly.

Reply:
reject changes

Reply:
This is easy to fix from within a proper editor environment, with the same core idea.

Reply:
what's problem?